### PR TITLE
Add Wakanim as hard

### PR DESCRIPTION
**Pull request by Minty123**

### CODE REVIEW

**Comment by Minty123** [comment]
It's not fully clear to me what's wrong with the formatting. I've copy pasted an entry from the existing list and validated it via JSON Lint. Maybe the GitHub Editor adds unwanted formatting for line breaks or sth. Will validate later.

File: _data/sites.json [hunk 1]
```diff
@@ -14916,6 +14916,18 @@
...
+        "url": "https://www.wakanim.tv/sc/v2/static/contactus",
+        "url_fr": "https://www.wakanim.tv/fr/v2/static/contactus",
+        "url_de": "https://www.wakanim.tv/de/v2/static/contactus",
+        "difficulty": "medium",
+        "notes": "Fill the form with your username and mail, select category \"Account/Login\" and subcategory \"Account Deletion\". An e-mail confirmation will be sent.",
+        "domains": [
+            "wakanim.tv"
+        ]
+    },
+
```
Comment (tupaschoal) [inline _data/sites.json:14930]:
```suggestion
    },

```

File: _data/sites.json [hunk 2]
```diff
@@ -14916,6 +14916,18 @@
         ]
     },
 
+    {
+        "name": "Wakanim",
+        "url": "https://www.wakanim.tv/sc/v2/static/contactus",
+        "url_fr": "https://www.wakanim.tv/fr/v2/static/contactus",
+        "url_de": "https://www.wakanim.tv/de/v2/static/contactus",
+        "difficulty": "medium",
+        "notes": "Fill the form with your username and mail, select category \"Account/Login\" and subcategory \"Account Deletion\". An e-mail confirmation will be sent.",
```
Comment (tupaschoal) [inline _data/sites.json:14925]: Is the e-mail confirmation just a note saying the account was deleted or something to take action to confirm the request?

**Review by tupaschoal — COMMENTED** [review]
> Maybe the GitHub Editor adds unwanted formatting for line breaks or sth. Will validate later.

Yes, it adds line breaks with whitespaces, as shown on my suggestion :\

File: _data/sites.json (hunk 2 above)
Comment (Minty123) [reply _data/sites.json]: Oh, great! Thanks so much for your help. :-) I've tried to make the notes a bit clearer. It's basically like opening a ticket. First it's just a note that they'll take care of it. Then there's another note that the request is completed.

Comment (tupaschoal) [reply _data/sites.json]: Ok, so then I think we should change it to hard, as it requires human intervention on the other side.

File: _data/sites.json [hunk 3]
```diff
@@ -14916,6 +14916,18 @@
         ]
     },
 
+    {
+        "name": "Wakanim",
+        "url": "https://www.wakanim.tv/sc/v2/static/contactus",
+        "url_fr": "https://www.wakanim.tv/fr/v2/static/contactus",
+        "url_de": "https://www.wakanim.tv/de/v2/static/contactus",
+        "difficulty": "medium",
```
Comment (tupaschoal) [inline _data/sites.json:14924]:
```suggestion
        "difficulty": "hard",
```

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**